Graph.py

```diff
@@ -42,9 +42,14 @@ def main():
         print('Test files loaded')
 
     """# Model"""
+    use_cuda = False
+    if args.gpu:
+        use_cuda = True
 
     model = select_model(args.model_def)
-    model = nn.DataParallel(model)
+    if use_cuda and torch.cuda.is_available():
+        model = model.cuda()
+        model = nn.DataParallel(model, device_ids=args.gpu_number)
 
```

Comment:
This line
` model = nn.DataParallel(model)`
 is still necessary if cuda is disabled.   The model has to be unwrapped.